index.css

```diff
@@ -701,3 +701,406 @@ body {
   margin-right: 20px;
 }
 
+@media screen and (min-width: 769px) {
```

Comment:
The layout of your main work section is also not as per the Figma template. Consider fixing it.
| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/33268927/177909601-0f4e19ef-d1f4-4d32-9efa-eb81b5032f61.png) | ![image](https://user-images.githubusercontent.com/33268927/177909663-cce50b6b-34f7-48e0-ab6f-feb267c80b3f.png) |